flink-examples/src/main/java/io/pravega/examples/flink/alert/AccessLog.java

```diff
@@ -10,22 +10,30 @@
  */
 package io.pravega.examples.flink.alert;
 
-import io.pravega.shaded.com.google.gson.Gson;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.core.JsonProcessingException;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.databind.ObjectMapper;
+import org.joda.time.DateTime;
 
 /**
  * Object to process Apache access log
  */
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class AccessLog {
     private String ClientIP;
```

Comment:
Could you please change the class member variable names to follow java bean standard naming convention?